easybuild/easyconfigs/p/pasta/pasta-1.8.5-GCC-8.2.0-2.31.1.eb

```diff
@@ -0,0 +1,41 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+easyblock = 'PythonPackage'
+
+name = 'pasta'
+version = '1.8.5'
+local_commit = '926a4eb'
+
+homepage = 'https://github.com/smirarab/pasta'
+description = """PASTA (Practical Alignment using SATe and Transitivity)"""
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
+
+github_account = 'smirarab'
+source_urls = [GITHUB_SOURCE, 'https://github.com/smirarab/sate-tools-linux/archive']
+sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ},
+           {'download_filename': '90fb074.tar.gz', 'filename': 'sate-tools-linux-90fb074.tar.gz'}]
+
+checksums = [
+    'e809562011266e2ed13a5ca4cf0accf9d62093f6885852828e07f3ebca325ac5',  # pasta-1.8.5.tar.gz
+    '48be3308d8f4e3bf92df528a1d6b00ad422376bedee12d4dcb9c1840f1f34f17',  # sate-tools-linux-90fb074.tar.gz
+]
+
+multi_deps = {'Python': ['3.7.2', '2.7.15']}
+
+skipsteps = ['build']
+
+dependencies = [
+    ('DendroPy', '4.4.0'),
+]
+
+preinstallopts = 'mv %(builddir)s/sate-tools-linux* ../sate-tools-linux && '
+install_target = 'develop'
+use_pip = False
+download_dep_fail = True
```

Comment:
@omula You should also add this to fix the shebang line in the Python scripts (which are hardcoded to one of Python versions):

```python
fix_python_shebang_for = ['bin/*.py']
```